.vscode/launch.json

```diff
@@ -6,7 +6,15 @@
     "configurations": [
 
         {
-            "name": "Run CraftBeerPi4",
+            "name": "setup CraftBeerPi4: create config folder structure",
+            "type": "python",
+            "request": "launch",
+            "module": "run",
+            "args": ["--config-folder-path=./.devcontainer/cbpi-dev-config", "setup"]
+        },
```

Comment:
:+1: I like that common tasks are placed in the available tasks in VSCode. That makes it easier to have an overview over what is frequently used in development. Also lets other start more easily. Il ike it!